docs/ExpectAPI.md

```diff
@@ -646,6 +647,45 @@ test('the house has my desired features', () => {
 });
 ```
 
+### `toHaveProperty(refKey, value)`
+
+Use `.toHaveProperty` to check if property at provided reference `refKey` exists for an object. 
+For checking deeply nested properties in an object use [dot notation](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Property_accessors) for deep references.
+
+You can also provide a value to check if it's strictly equal to the `value` present
+at `refKey` on the target object.
+
+The following example contains a `houseForSale` object with nested properties. We are using `toHaveProperty`
+to check for the existence and values of some properties in the object.
+```js
+// Object containing house features to be tested
+const houseForSale = {
+	bath: true,
+	kitchen: {
+		amenities: ['oven', 'stove', 'washer'],
+		area: 20,
+		wallColor: 'white'
+	},
+  bedrooms: 4
```

Comment:
i think the indentation is off here.
at facebook we also use trailing commas. do you mind adding one? :)
```js
const houseForSale = {
  // ...
  bedrooms: 4,
};
```